st2.json

```diff
@@ -1,137 +1,152 @@
 {
-  "variables": {
-    "purpose": "Packer template for building StackStorm Vagrant '.box' & '.ova' Virtual Appliance",
-    "st2_version": "{{env `ST2_VERSION`}}",
-    "box_version": "{{env `BOX_VERSION`}}",
-    "st2_user": "st2admin",
-    "st2_password": "Ch@ngeMe",
-    "vm_name": "st2",
-    "vm_description": "Box with StackStorm (aka 'IFTTT for Ops') event-driven automation platform: auto-remediation, security responses, facilitated troubleshooting, complex deployments, ChatOps and more. \n* https://stackstorm.com/ \n* Documentation: https://docs.stackstorm.com/ \n* Community: https://stackstorm.com/community-signup \n* Forum: https://forum.stackstorm.com/"
-  },
```

Comment:
Can't remember exactly, I think it was a (more recent) version of Packer that was failing validation because the keys weren't lexicographically sorted.

Reply:
Hrm, scratch what I just said, variables at the top is passing `packer validate` (I'll restore).